CRISPResso2/CRISPRessoReports/templates/report.html

```diff
@@ -635,6 +632,25 @@ <h5>Base editing for {{amplicon_name}}</h5>
                     					</div>
 										{% endfor %}
 									</div>
+                  {% if amplicon_name in report_data['figures']['sgRNA_based_names'] and '10h' in report_data['figures']['sgRNA_based_names'][amplicon_name] %}
+									<div class="tab-pane fade" id="{{amplicon_name}}_10h" role="tabpanel">
+                    {% for fig_name in report_data['figures']['sgRNA_based_names'][amplicon_name]['10h'] %}
+                      <div class='mb-4'>
+                      {{ render_partial('shared/partials/fig_reports.html', report_data=report_data, fig_name=fig_name, amplicon_name=amplicon_name)}}
```

Comment:
I believe that we should add `amplicon_name` to `fig_name` here, because the `fig_name` becomes the `id` for the `div` and `fig_name` is not unique across amplicons. So for each amplicon, there are multiple `div`'s with the `id='fig_plot_10h_0` (see https://github.com/edilytics/CRISPResso2_tests/pull/64/files#diff-17833c0759af6c77c8a552ad03fc8ee2e94a1e96d2d31f8a0e33c0ba98b279e0R1533).